tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/Configuration/ProjectConfigurationDimensionProviderTestBase.cs

```diff
@@ -569,9 +569,9 @@ private static void AssertDefaultOrEmpty(BaseProjectConfigurationDimensionProvid
         {
             if (provider.DimensionDefaultValue is not null)
             {
-                Assert.Single(result);
-                Assert.Equal(provider.DimensionName, result.First().Key);
-                Assert.Equal(provider.DimensionDefaultValue, result.First().Value);
+                (string key, string value) = Assert.Single(result);
+                Assert.Equal(provider.DimensionName, key);
```

Comment:
Line 564 has similar thing? At least it can use result of Assert.Single rather than calling First.